cmake/binarytochararray.cmake

```diff
@@ -0,0 +1,51 @@
+# Copyright libCellML Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.cmake_minimum_required (VERSION 3.1)
+
+# This script requires CMake version 3.18 to run hence why it is not directly part of the build configuration.
+# usage:
+#  cmake -P binary_to_char_array.cmake <full-path-to-xhtml-math11-f.dtd>
```

Comment:
Not needed since we can see that from `~/CMakeLists.txt`?

Reply:
I think it is needed as a reminder to how it should be when the minimum version allows.

Reply:
Fine.